setup.py

```diff
@@ -61,6 +61,7 @@ def get_file_text(file_name):
         "chalice": ["chalice>=1.16.0"],
         "httpx": ["httpx>=0.16.0"],
         "starlette": ["starlette>=0.19.1"],
+        "starlite": ["starlite>=1.39.0"],
```

Comment:
can be updated to a later version I think

Reply:
latest?

Reply:
1.48.1